sw_watchdog_msgs/package.xml

```diff
@@ -16,8 +16,6 @@
 
   <test_depend>ament_lint_auto</test_depend>
   <test_depend>ament_lint_common</test_depend>
-  <test_depend>ros_testing</test_depend>
```

Comment:
What specifically is being tested?

Reply:
I removed `ros_testing` from the dependencies because there were no node tests in sw_watchdog_msgs. I think it won't test much for this package except standard things like manifest and copyright.